sources/api/migration/migrations/v1.12.0/oci-defaults-setting/src/main.rs

```diff
@@ -0,0 +1,25 @@
+#![deny(rust_2018_idioms)]
+
+use migration_helpers::common_migrations;
+use migration_helpers::{migrate, Result};
+use std::process;
+
+/// We added new settings for configuring the default OCI runtime spec,
+/// `settings.oci-defaults`, which will initially contain
+/// `settings.oci-defaults.capabilities` and
+/// `settings.oci-defaults.resource-limits`
+fn run() -> Result<()> {
+    migrate(common_migrations::AddPrefixesMigration(vec![
+        "settings.oci-defaults",
+    ]))
+}
+
+// Returning a Result from main makes it print a Debug representation of the error, but with Snafu
+// we have nice Display representations of the error, so we wrap "main" (run) and print any error.
+// https://github.com/shepmaster/snafu/issues/110
+fn main() {
+    if let Err(e) = run() {
+        eprintln!("{}", e);
+        process::exit(1);
+    }
+}
```

Comment:
Need to look at an example for platform filtering -- only want to do migration on K8S variants

Reply:
https://github.com/bottlerocket-os/bottlerocket/blob/c6456aeea38e284304557e03c9062501169b61db/sources/api/migration/migrations/archived/v1.8.0/pki-affected-services/src/main.rs#L18